pt/lessons/basics/strings.md

```diff
@@ -0,0 +1,151 @@
+---
+layout: page
+title: Strings
+category: basics
+order: 14
+lang: pt
+---
+
+Strings, listas de caracteres, Graphemes e Codepoints.
+
+{% include toc.html %}
+
+## Strings
+
+Strings no Elixir são nada mais que uma sequência de bytes. Vamos ver um exemplo:
```

Comment:
Hi @ruan-brandao! I took the liberty to review and add some notes...

Maybe "Strings **em** Elixir..."?